From 85151a05841f93deab2e5c673c8d732fb3c47100 Mon Sep 17 00:00:00 2001
From: Tianshu Qiu <tian.shu.qiu@intel.com>
Date: Tue, 15 May 2018 14:35:01 +0800
Subject: [PATCH 1920/2345] media: intel-ipu4: imx355: Enable ACPI registration

Message for Open Source:
Enable ACPI registration

Message for Internal:
[Issue/Feature] ACPI is not enabled.
[Root Cause/Changes] Enable ACPI registration

Change-Id: I734e998182d07539d85dfe2539338372164759d9
Signed-off-by: Tianshu Qiu <tian.shu.qiu@intel.com>
Signed-off-by: Meng Wei <wei.meng@intel.com>
---
 drivers/media/i2c/imx355.c | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/drivers/media/i2c/imx355.c b/drivers/media/i2c/imx355.c
index 5e4b149..fb52416 100644
--- a/drivers/media/i2c/imx355.c
+++ b/drivers/media/i2c/imx355.c
@@ -1792,10 +1792,9 @@ static const struct dev_pm_ops imx355_pm_ops = {
 
 #ifdef CONFIG_ACPI
 static const struct acpi_device_id imx355_acpi_ids[] = {
-	{ "IMX355" },
+	{"SONY355A"},
 	{ /* sentinel */ }
 };
-
 MODULE_DEVICE_TABLE(acpi, imx355_acpi_ids);
 #endif
 
@@ -1804,7 +1803,9 @@ static struct i2c_driver imx355_i2c_driver = {
 		.name = "imx355",
 		.owner = THIS_MODULE,
 		.pm = &imx355_pm_ops,
-		/*.acpi_match_table = ACPI_PTR(imx355_acpi_ids),*/
+#ifdef CONFIG_ACPI
+		.acpi_match_table = ACPI_PTR(imx355_acpi_ids),
+#endif
 	},
 	.probe = imx355_probe,
 	.remove = imx355_remove,
-- 
2.7.4

